src/gluonts/evaluation/backtest.py

```diff
@@ -179,49 +159,9 @@ def backtest_metrics(
     else:
         logger = logging.getLogger(__name__)
 
-    if train_dataset is not None:
-        train_statistics = calculate_dataset_statistics(train_dataset)
-        serialize_message(logger, train_dataset_stats_key, train_statistics)
-
     test_statistics = calculate_dataset_statistics(test_dataset)
     serialize_message(logger, test_dataset_stats_key, test_statistics)
 
-    if isinstance(forecaster, Estimator):
-        serialize_message(logger, estimator_key, forecaster)
-        assert train_dataset is not None
-
-        predictor = (
-            forecaster.train(
-                train_dataset,
-                num_workers=num_workers,
-                num_prefetch=num_prefetch,
-                **kwargs,
-            )
-            if isinstance(forecaster, GluonEstimator)
-            else (forecaster.train(train_dataset))
-        )
-
-        if isinstance(forecaster, GluonEstimator) and isinstance(
-            predictor, GluonPredictor
-        ):
-            inference_data_loader = InferenceDataLoader(
-                dataset=test_dataset,
-                transform=predictor.input_transform,
-                batch_size=forecaster.trainer.batch_size,
-                ctx=forecaster.trainer.ctx,
-                dtype=forecaster.dtype,
-            )
-
-            if forecaster.trainer.hybridize:
-                predictor.hybridize(batch=next(iter(inference_data_loader)))
-
-            if use_symbol_block_predictor:
-                predictor = predictor.as_symbol_block_predictor(
-                    batch=next(iter(inference_data_loader))
-                )
-    else:
-        predictor = forecaster
-
     forecast_it, ts_it = make_evaluation_predictions(
         test_dataset, predictor=predictor, num_samples=num_samples
     )
```

Comment:
Note that now the role of `backtest_metrics` is *essentially* just to call this here, plus run the evaluator through the results (two lines below). This makes you think: do we need `backtest_metrics` at all?